pro/src/screens/OfferIndividual/StocksEvent/StocksEvent.tsx

```diff
@@ -115,28 +115,44 @@ const StocksEvent = ({ offer }: IStocksEventProps): JSX.Element => {
     initialValues,
     onSubmit,
     validationSchema: getValidationSchema(minQuantity),
+    // enableReinitialize is needed to reset dirty after submit (and not block after saving a draft)
```

Comment:
mostly needed to reinitialize the form after initialValue update and so not submiting duplicated stocks when clicking multiple times on "Save draft"

Reply:
both of them, je rajoute !